### DISCUSSION TITLE
Delete all old data after HPOS sync

### DISCUSSION THREAD

**Discussion by clonerdev** [woocommerce/woocommerce#41095] [WooCommerce HPOS]
Thank you very much for the possibility of HPOS, the problem is that after activating HPOS and after all data is synchronized and synchronization is disabled, it is possible to delete the data in the old _post and _postmeta tables and... It doesn't exist, even though they are no longer needed and they only make the database heavy and other things.
Please add the possibility to delete old data that you don't need after complete synchronization.
Also, the possibility of checking can be added so that before deleting the old data or after activating and synchronizing HPOS, the data of the old tables are checked with the new tables and the result is declared that nothing has been missed.
* What I mean by deleting the old data left behind is all the data that has been transferred and the old data is no longer needed, orders, products, etc.
Thanks

**Comment by rrennick** [comment]
Closing this as a duplicate of https://github.com/woocommerce/woocommerce/discussions/40988.
